Dockerfile

```diff
@@ -1,13 +1,28 @@
 # base image
-FROM node:12.2.0-alpine
+# Vite did not like the older version of Node that we'd specified, so I switched to the most recent LTS version
+FROM node:hydrogen-alpine
```

Comment:
Looks good to me, we just need to take note of the node engine running on the deployment environment as well.